openassessment/table_prompts/apps.py

```diff
@@ -0,0 +1,8 @@
+# -*- coding: utf-8 -*-
+from __future__ import unicode_literals
+
+from django.apps import AppConfig
+
+
+class MobileAssessmentConfig(AppConfig):
```

Comment:
As we have renamed the app let's please rename this config class as well.

Reply:
Done